src/schema/generated-schema-types.ts

```diff
@@ -1480,6 +1480,13 @@ export type ExternalAccountTestResponse = {
   success: Scalars['Boolean'];
 };
 
+export enum ExternalSendersType {
+  All = 'ALL',
```

Comment:
zimbraPrefExternalSendersType LDAP attribute is accepting values in CAPS.

I have attached the code snippet for your reference.

```
<attr id="1320" name="zimbraPrefExternalSendersType" type="enum" value="ALL,ALLNOTINAB,INAB,INSD" cardinality="single" optionalIn="account,cos" flags="accountInherited" since="8.0.0">
  <defaultCOSValue>ALL</defaultCOSValue>
  <desc>Specifies the meaning of an external sender.  "ALL" means users whose domain doesn't match the recipient's or zimbraInternalSendersDomain.  "ALLNOTINAB" means "ALL" minus users who are in the recipient's address book.
  "INAB" Users/Addresses whose domain doesn't match the recipient's domain or zimbraInternalSendersDomain and which are present in recipient's address book.
  "INSD" means users whose domain matches the specific domain</desc>
</attr>
```

Reply:
that's about values, I am talking about keys

Reply:
These keys are automatically generated, so they cannot be camelcased. So we have to use those keys as they are.

Reply:
These keys are automatically generated, based on the defined values in the schema. Hence we cannot modify the keys.